### ISSUE TITLE
Add shorter alias for `env_var` functions

### ISSUE THREAD

**Issue by casey** [casey/just#1278]
These are nice and legible, but a little too long, since they're very common.

Some ideas:

Add a `$` operator which is a prefix operator that dereferences its argument:

```
export PATH := "/foo/bin:" + $"PATH"
```

Add a shorter function:

```
export PATH := "/foo/bin:" + env("PATH")
```

I like the `$` operator, since why not, but I know that I have an extremely high tolerance for line-noise-like syntax.

**Comment by runeimp** [comment]
I like both options. The only downside is the `$"VARNAME"` is very close to the standard POSIX style `$VARNAME` or `${VARNAME}` that everyone using such systems will flub by typing `"$VARNAME"` instead like 98.643% of the time. Yes, that _exact_ amount of the time.  😉 

So I think just adopting standard Bourne shell / POSIX syntax for variables would be _much_ better. Especially if their to be specifically used to access actual environment variables and not internal Justfile variables or some such.

The other downside is that Windows users will either prefer `$Env:VARNAME` or `%VARNAME%` style syntax. And those that are semi-familiar with Bourne shell / POSIX syntax will 99.994% 👼 of the time use the wrong syntax as well.

Oh and don't come back with 'My version of Zsh accepts $"VARNAME" syntax.' because I don't believe it. 😉

Also `env(VARNAME)` is more neutral and stands out as part Just rather than potentially being somehow possibly part of the shell Just is running in. 😁

**Comment by casey** [comment]
I feel you. `$"FOO"` is pretty ugly, and you're right it's not familiar so people will mess it up. `$FOO` is, if you think about it though, pretty weird. `FOO` is usually an identifier that points to a variable, but in this case it's a special way of writing the string `"FOO"`, i.e. the identifier is just interpreted as a bare word string. `$"FOO"` is slightly more flexible. But maybe if the point is to have short syntax, it should be `$FOO`. I think `env` is good too. I feel like `env_var` should be deprecated in favor of `env` whatever we decide about `$`.

**Comment by nk9** [comment]
FWIW, I prefer `env()` and also really don't like `$"FOO"`.

**Comment by casey** [comment]
I think we should add `env` as an alias, since it's obviously better than `env_var`. We could also `env(NAME, DEFAULT)` to replace `env_or_default(NAME, DEFAULT)`. `$"FOO"` is probably just too weird.

**Comment by nk9** [comment]
Yeah, just having "env" do both seems like the right move.

**Comment by kykyi** [comment]
@casey can I open a PR for this issue? Would be a great first one 👌

**Comment by nk9** [comment]
Not sure how available Casey is for reviews at the moment, but the work will eventually need to be done. I would say go for it, hopefully it won't take too long. There may be a bit of a delay before it can be merged, though.

**Comment by kykyi** [comment]
@nk9 and Casey when you get around to it, here is a PR adding an `env` alias 😄 

https://github.com/casey/just/pull/1613

**Comment by Adda0** [comment]
I believe that this issue has been resolved in #1613 (great job, btw.; `env(VAR, DEFAULT)` looks pretty neat) and should be therefore closed as such. That is, unless you plan to act on 
>  I feel like env_var should be deprecated in favor of env

as a task to be resolved before closing this issue.
